src/core/packages/chrome/browser-internal/src/chrome_service.tsx

```diff
@@ -494,48 +544,38 @@ export class ChromeService {
           return <ProjectHeaderWithNavigationComponent />;
         }
 
-        return (
-          <Header
-            isServerless={this.isServerless}
-            loadingCount$={http.getLoadingCount$()}
-            application={application}
-            headerBanner$={headerBanner$.pipe(takeUntil(this.stop$))}
-            badge$={badge$.pipe(takeUntil(this.stop$))}
-            basePath={http.basePath}
-            breadcrumbs$={breadcrumbs$.pipe(takeUntil(this.stop$))}
-            breadcrumbsAppendExtensions$={breadcrumbsAppendExtensions$.pipe(takeUntil(this.stop$))}
-            customNavLink$={customNavLink$.pipe(takeUntil(this.stop$))}
-            kibanaDocLink={docLinks.links.kibana.guide}
-            docLinks={docLinks}
-            forceAppSwitcherNavigation$={navLinks.getForceAppSwitcherNavigation$()}
-            globalHelpExtensionMenuLinks$={globalHelpExtensionMenuLinks$}
-            helpExtension$={helpExtension$.pipe(takeUntil(this.stop$))}
-            helpSupportUrl$={helpSupportUrl$.pipe(takeUntil(this.stop$))}
-            helpMenuLinks$={helpMenuLinks$}
-            homeHref={http.basePath.prepend('/app/home')}
-            kibanaVersion={injectedMetadata.getKibanaVersion()}
-            navLinks$={navLinks.getNavLinks$()}
-            recentlyAccessed$={recentlyAccessed.get$()}
-            navControlsLeft$={navControls.getLeft$()}
-            navControlsCenter$={navControls.getCenter$()}
-            navControlsRight$={navControls.getRight$()}
-            navControlsExtension$={navControls.getExtension$()}
-            onIsLockedUpdate={setIsNavDrawerLocked}
-            isLocked$={getIsNavDrawerLocked$}
-            customBranding$={customBranding$}
-          />
-        );
+        return getClassicHeader({ isFixed: true, includeBanner: true, as: 'header' });
       };
 
       return <HeaderComponent />;
     };
 
+    const getClassicHeaderComponentForGridLayout = () => {
```

Comment:
nit: `ChromeService` has been driving me crazy with all the methods it contains, apparently scoped to different use cases.

It may not be germane to _this_ PR, but we've got to find some time to clean this thing up... or at the very least, split it up somehow.  Perhaps make this an abstract class with use-case implementations?

Again: I think something for later.  But I'm resigned to see another pair of use-case specific, complex-named methods added to this service.